### PR TITLE
[HTTPCORE-572] Move examples to the src/test folders for each module.

### PR DESCRIPTION
**Pull request by garydgregory**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -185,7 +185,15 @@
               <outputDirectory>${basedir}/target/site/examples</outputDirectory>
               <resources>
                 <resource>
-                  <directory>src/examples</directory>
+                  <directory>src/test/java/org/apache/hc/core5/http/examples</directory>
```
Comment (ok2c) [inline pom.xml]: @garydgregory May I suggest that `maven-resources-plugin` section be moved to individual module poms? Module specific stuff in the main project pom looks a bit dodgy now.

Comment (garydgregory) [reply pom.xml]: @ok2c ,

Thank you for the review. I committed your requested changes.

Gary

**Comment by ok2c** [comment]
@garydgregory Looks good. Could you please squash the commit though?

File: pom.xml (hunk 1 above)
Comment (garydgregory) [reply pom.xml]: I must have messed up the squashing. I think it is easier to squash in GH when you merge. Can we do it that way?

**Pull request merged**